content/providers/migration.md

```diff
@@ -19,8 +19,184 @@ Documentation coming soon.
+plan. It prompts for all required options when executing the command.
+
+#### Required inputs
+
+When running the `family-migrator generate` command, the tool prompts the
+user with a series of questions to gather the required inputs:
+
+<!-- vale Google.Will = NO -->
+<!-- vale gitlab.FutureTense = NO -->
+<!-- vale write-good.Passive = NO -->
+{{< table "table table-sm table-striped">}}
+| Required input | Description |
+| ----- | ----------- |
+| **Please specify the path for the migration plan** | The answer represents the output path of the migration plan. Users need to provide a path for generating the plan and patch files. For example, `/tmp/generated/migration-plan.yaml` |
+| **Please provide the registry and organization for the provider family packages** | The format must be `<registry host>/<organization>`, for example, `xpkg.upbound.io/upbound`. |
+| **Please select the providers that will be migrated** | The tool presents user with a list of the monolithic providers they can select to migrate from. The tool supports multiple selections. |
+| **Please specify the version of the provider-abc family** | Provide the versions of the family provider to migrate to for each selected provider. If you want to migrate to the latest version, refer to the [Upbound Marketplace](https://marketplace.upbound.io). The version format must be `v0.y.y` |
+| **Please enter the URL of the migration source Configuration package** | Provide the registry URL of the currently installed Configuration Package. The value appears in the `spec.package` path of `Configuration.pkg.crossplane.io/v1` resource. For example, `xpkg.upbound.io/upbound/platform-ref-gcp:v0.3.0` |
+| **Please enter the URL of the migration target Configuration package** | The tool builds a new configuration package during migration. Specify the registry URL where to update the built package. For example, `<registry-host>/<organization>/platform-ref-gcp:v0.3.0-migrated` |
+| **Please specify the source directory for the Crossplane Configuration package** | Specify the source directory to the location of the configuration package. This directory contains the Configuration.meta.pkg.crossplane.io manifest that contains the provider dependencies, Compositions, and CompositeResourceDefinitions. For example: `/Users/user/workspace/platform-ref-gcp/package` Example configuration package: https://github.com/upbound/platform-ref-gcp/tree/main/package. |
+| **Please specify the path to the directory containing the Crossplane package examples** | Specify the directory to the location of the configuration package examples. This directory may contain Claims, and Configuration.pkg.crossplane.io resource. For example: `/Users/user/workspace/platform-ref-gcp/examples` Example configuration examples: https://github.com/upbound/platform-ref-gcp/tree/main/examples. |
+| **Please specify the path to store the updated configuration package** | The location to store the newly built configuration package. |
```

Comment:
Can't we just re-use the path from step 1?

Reply:
We started to use it in this [PR](https://github.com/upbound/extensions-migration/pull/19). I removed this.